libindy/ci/amazon.dockerfile

```diff
@@ -29,7 +29,14 @@ RUN cd /tmp && \
 ENV PKG_CONFIG_PATH=$PKG_CONFIG_PATH:/usr/local/lib/pkgconfig
 ENV LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib
 
-ENV RUST_ARCHIVE=rust-1.19.0-x86_64-unknown-linux-gnu.tar.gz
+RUN yum install -y java-1.8.0-openjdk-devel
+ENV JAVA_HOME /usr/lib/jvm/java-1.8.0-openjdk
+
+RUN wget https://repos.fedorapeople.org/repos/dchen/apache-maven/epel-apache-maven.repo -O /etc/yum.repos.d/epel-apache-maven.repo
```

Comment:
@vimmerru there is alternative option to manual download from maven.apache.org ZIP archive, unzip and set PATH... What option do you prefer?

Reply:
I more like usage of fedora repo.